benchmark_runner/grafana/perf/jsonnet/main.libsonnet

```diff
@@ -225,7 +225,7 @@ g.dashboard.new('PerfCI-Regression-Summary')
10170": {"index": 25, "text": "4.10.1-70"}, "410197": {"index": 26, "text": "4.10.1-97"}, "411115": {"index": 52, "text": "4.11.1-15"}, "411121": {"index": 51, "text": "4.11.1-21"}, "411135": {"index": 46, "text": "4.11.1-35"}, "411142": {"index": 47, "text": "4.11.1-42"}, "411605": {"index": 53, "text": "4.11.6-5"}, "412116": {"index": 74, "text": "4.12.1-16"}, "412119": {"index": 84, "text": "4.12.1-19"}, "412122": {"index": 75, "text": "4.12.1-22"}, "412139": {"index": 82, "text": "4.12.1-39"}, "412140": {"index": 81, "text": "4.12.1-40"}, "412317": {"index": 121, "text": "4.12.3-17"}, "413014": {"index": 101, "text": "4.13.0-ec.4"}, "413118": {"index": 113, "text": "4.13.1-18"}, "413140": {"index": 114, "text": "4.13.1-40"}, "4100683": {"index": 17, "text": "4.10.0-683"}, "4100688": {"index": 18, "text": "4.10.0-688"}, "4100700": {"index": 19, "text": "4.10.0-700"}, "4100729": {"index": 20, "text": "4.10.0-729"}, "4101101": {"index": 27, "text": "4.10.1-101"}, "4110137": {"index": 50, "text": "4.11.0-137"}, "4120173": {"index": 83, "text": "4.12.0-173"}, "4120777": {"index": 71, "text": "4.12.0-777"}, "4120781": {"index": 72, "text": "4.12.0-781"}, "4131154": {"index": 115, "text": "4.13.1-154"}, "41301586": {"index": 88, "text": "4.13.0-1586"}, "41301649": {"index": 89, "text": "4.13.0-1649"}, "41301666": {"index": 91, "text": "4.13.0-1666"}, "41301689": {"index": 90, "text": "4.13.0-1689"}, "41301782": {"index": 92, "text": "4.13.0-1782"}, "41301856": {"index": 93, "text": "4.13.0-1856"}, "41301938": {"index": 94, "text": "4.13.0-1938"}, "41301943": {"index": 95, "text": "4.13.0-1943"}, "41302115": {"index": 96, "text": "4.13.0-2115"}, "41302176": {"index": 97, "text": "4.13.0-2176"}, "41302229": {"index": 98, "text": "4.13.0-2229"}, "41302251": {"index": 99, "text": "4.13.0-2251"}, "41302269": {"index": 100, "text": "4.13.0-2269"}, "4130ec3": {"index": 87, "text": "4.13.0-ec.3"}, "CNV": {"index": 112, "text": "CNV"}, "KATA": {"index": 116, "text": "KATA"}, "OCP": {"index": 108, "text": "OCP"}, "ODF": {"index": 120, "text": "ODF"}, "Product Versions": {"index": 107, "text": "Product Versions"}, "4136": {"index": 122, "text": "4.13.6"}, "413360": {"index": 123, "text": "4.13.3-60"}, "41244": {"index": 126, "text": "4.12.4-4"}, "4134": {"index": 130, "text": "4.13.4"}, "4131206": {"index": 131, "text": "4.13.1-206"}, "4131214": {"index": 132, "text": "4.13.1-214"}, "41323": {"index": 133, "text": "4.13.2-3"}, "413273": {"index": 134, "text": "4.13.2-73"}, "41338": {"index": 135, "text": "4.13.3-8"}, "413267": {"index": 136, "text": "4.13.2-67"}, "4137": {"index": 137, "text": "4.13.7"}, "4133166": {"index": 138, "text": "4.13.3-166"}, "141": {"index": 139, "text": "1.4.1"}, "41252": {"index": 140, "text": "4.12.5-2"}, "4138": {"index": 141, "text": "4.13.8"}, "4133203": {"index": 142, "text": "4.13.3-203"}, "4139": {"index": 143, "text": "4.13.9"}, "413426": {"index": 145, "text": "4.13.4-26"}, "4133266": {"index": 146, "text": "4.13.3-266"}}
```

Comment:
This line is generated auto by the code but you can find the added code at the end of the line.
I dont see any reason to put in in several lines

Reply:
If it's auto-generated, then it's OK.  If it weren't, it would be easier to compare (diff tools like to compare lines).  But since it's autogenerated, it's fine as is.